### PR TITLE
GLVis 4.0 Support

### PR DESCRIPTION
**Pull request by tomstitt**
I had to work around a fetch issue with `libpng` on LC but I've tested this with GLVis@4.0 on toss3 and OS X

To build `glvis@develop` on os x you may need a newer sdl2 (in another PR https://github.com/spack/spack/pull/25046)

### CODE REVIEW

**Review by tzanio — DISMISSED** [review]
Thank you so much for adding this @tomstitt !

File: var/spack/repos/builtin/packages/glvis/package.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Glvis(MakefilePackage):
     homepage = "http://glvis.org"
     git      = "https://github.com/glvis/glvis.git"
 
-    maintainers = ['goxberry', 'v-dobrev', 'tzanio']
+    maintainers = ['goxberry', 'v-dobrev', 'tzanio', 'stitt4']
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/glvis/package.py]: Apparently there's no github user with this name...

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/glvis/package.py (hunk 1 above)
Comment (tzanio) [reply var/spack/repos/builtin/packages/glvis/package.py]: Pretty sure that should be `tomstitt` 😁

Comment (tomstitt) [reply var/spack/repos/builtin/packages/glvis/package.py]: oops! yes that is not my github name..

edit: fixed

**Review by alalazo — APPROVED** [review]
@tzanio approved already

**Pull request merged**